diff -ur iguanaIR-1.0pre2.orig//reflasher/Makefile iguanaIR-1.0pre2/reflasher/Makefile
--- iguanaIR-1.0pre2.orig//reflasher/Makefile	2009-11-05 02:17:46.000000000 +0100
+++ iguanaIR-1.0pre2/reflasher/Makefile	2010-10-09 12:55:27.249923736 +0200
@@ -10,17 +10,17 @@
 	ln -s ../../../../firmware/usb_ir.loader/output/usb_ir.hex hex/loader-0.hex
 
 install:
-	install -d $(DESTDIR)$(prefix)/lib/$(NAME)/hex
-	install $(NAME) $(DESTDIR)$(prefix)/lib/$(NAME)
-	install $(filter-out hex/loader-0.hex hex/body-0.hex, $(wildcard hex/*.hex)) $(DESTDIR)$(prefix)/lib/$(NAME)/hex
+	install -d $(DESTDIR)$(prefix)/lib64/$(NAME)/hex
+	install $(NAME) $(DESTDIR)$(prefix)/lib64/$(NAME)
+	install $(filter-out hex/loader-0.hex hex/body-0.hex, $(wildcard hex/*.hex)) $(DESTDIR)$(prefix)/lib64/$(NAME)/hex
 	install -d $(DESTDIR)$(prefix)/bin
 	rm -f $(DESTDIR)$(prefix)/bin/$(NAME)
-	ln -s ../lib/$(NAME)/$(NAME) $(DESTDIR)$(prefix)/bin/$(NAME)
+	ln -s ../lib64/$(NAME)/$(NAME) $(DESTDIR)$(prefix)/bin/$(NAME)
 
 uninstall:
 	rm -f $(DESTDIR)$(prefix)/bin/$(NAME)
-	rm -f $(DESTDIR)$(prefix)/lib/$(NAME)/hex/*.hex
-	rmdir $(DESTDIR)$(prefix)/lib/$(NAME)/hex 2>/dev/null || true
-	rm -f $(DESTDIR)$(prefix)/lib/$(NAME)/upgrade-usb
+	rm -f $(DESTDIR)$(prefix)/lib64/$(NAME)/hex/*.hex
+	rmdir $(DESTDIR)$(prefix)/lib64/$(NAME)/hex 2>/dev/null || true
+	rm -f $(DESTDIR)$(prefix)/lib64/$(NAME)/upgrade-usb
 
 .PHONY: all rpm dev devel tarball install
