--- iguanaIR-1.1.0/CMakeLists.txt.orig	2013-07-28 19:08:08.000000000 +0200
+++ iguanaIR-1.1.0/CMakeLists.txt	2015-10-05 16:07:49.326406232 +0200
@@ -68,7 +68,7 @@
       Set(DAEMONLIBS pthread dl)
     EndIf()
 
-    Set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O2 -Wall -pedantic")
+    Set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -pedantic")
 
     # see if we have clock_gettime in time.h
     Set(CMAKE_REQUIRED_LIBRARIES rt)
