--- iguanaIR-1.1.0/CMakeLists.txt.orig	2015-10-05 16:47:55.412971926 +0200
+++ iguanaIR-1.1.0/CMakeLists.txt	2015-10-05 17:03:10.749600180 +0200
@@ -97,6 +97,7 @@
 add_library(iguanaIR SHARED
                      iguanaIR.c dataPackets.c ${PIPESRC}
                      ${BASESRC})
+	     set_target_properties(iguanaIR PROPERTIES SOVERSION 0)
 set_property(TARGET iguanaIR
              APPEND PROPERTY COMPILE_DEFINITIONS IGUANAIR_EXPORTS)
 install(TARGETS iguanaIR
