--- iguanair-9336f121b4127f4ac494e5b26b82ce9c6b86a0ac/software/usb_ir/CMakeLists.txt.orig	2017-12-22 16:26:48.778304231 +0100
+++ iguanair-9336f121b4127f4ac494e5b26b82ce9c6b86a0ac/software/usb_ir/CMakeLists.txt	2017-12-22 16:52:15.058286803 +0100
@@ -62,7 +62,6 @@
   Set(DAEMONSRC daemon.c)
   List(APPEND BASESRC compat-unix.c)
   Set(CMAKE_REQUIRED_FLAGS "-I/usr/include")
-  add_c_flag(-pedantic -g -O2)
 
   # see if we have clock_gettime possibly in rt
   check_library_exists(rt clock_gettime "time.h" HAVE_CLOCK_GETTIME)
